graphics/blender/Portfile

```diff
@@ -248,6 +248,27 @@ list(APPEND USD_LIBRARIES\\
             ${platform_apple.cmake}
     }
 
+    # Fix hard-coded paths in the Find*.cmake scripts
+    foreach f [glob -directory \
+        ${worksrcpath}/build_files/cmake/Modules *.cmake \
+    ] {
+        foreach re [list \
+            "s|/usr/local|${prefix}|" \
+            "s|/usr/lib|${prefix}/lib|" \
+            "s|/usr/include|${prefix}/include|" \
+            "s|/usr/share|${prefix}/share|" \
+            "s|/opt/lib/\[0-9A-Za-z\]+|${prefix}/lib|" \
+            "s|/opt/include|${prefix}/include|" \
+            "s|/opt/share|${prefix}/share|" \
+        ] {
+            reinplace -q -E $re $f
+        }
+    }
+    reinplace -E "s|(${prefix}/lib)|\\1/opencollada|" \
+        ${worksrcpath}/build_files/cmake/Modules/FindOpenCOLLADA.cmake
+    reinplace -E "s|(${prefix}/lib)|\\1/usd|" \
+        ${worksrcpath}/build_files/cmake/Modules/FindUSD.cmake
+
```

Comment:
It's a bit of a hammer, but I imagine it works. I assume really is no way to explicitly set these locations during configuration?

Reply:
> It's a bit of a hammer, but I imagine it works. I assume really is no way to explicitly set these locations during configuration?

It's possible, but about as equal of a hack as what I'm doing. The upstream devs make it easy to turn the feature completely off by using `-DWITH_USD=OFF`, but the source code generally relies on the Find*.cmake scripts to figure out the paths. MacPorts' cmake PortGroup does a pretty good job of providing hints to CMake in terms of trying to tell it to look in `/opt/local`, but it does occasionally miss certain locations.